lib/server.js

```diff
@@ -286,6 +289,10 @@ function getServerInstance (options, httpHandler) {
       this.close()
     }))
 
+    if (options.forceCloseConnections === true) {
```

Comment:
Can you add a note that states that this is needed for Node.js versions prior to v24?